shell/common/api/electron_api_native_image.cc

```diff
@@ -62,6 +62,7 @@ base::FilePath NormalizePath(const base::FilePath& path) {
     return path;
   }
 
+  base::ThreadRestrictions::ScopedAllowIO allow_blocking;
   base::FilePath absolute_path = MakeAbsoluteFilePath(path);
```

Comment:
Just noticed this, did we qualify `base` somewhere globally ?

Reply:
It seems to be augment-dependent lookup.